server/socket/socketManager.js

```diff
@@ -76,7 +76,17 @@ exports.socketManager = (io, socket) => {
     }
   });
 
-  socket.on('updatedRobotJobStatus', ({ jobId, status }) => {
-    socketHelperFunctions.updateRobotJobStatus(jobId, status);
+  socket.on('updatedRobotJob', ({ jobId, robotLogs }) => {
```

Comment:
I do not quite get the naming here. The event is called 'updatedRobotJob', a past wording, while this function then updates the robot job? Please think of a better way of naming either the event or the called function

Reply:
Good point :)